src/Microsoft.Azure.WebJobs.Host/Blobs/Triggers/BlobTriggerAttributeBindingProvider.cs

```diff
@@ -18,14 +18,65 @@
 using Microsoft.Azure.WebJobs.Host.Triggers;
 using Microsoft.Extensions.Logging;
 using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.Azure.WebJobs.Host.Config;
+using System.Threading;
 
 namespace Microsoft.Azure.WebJobs.Host.Blobs.Triggers
 {
+    internal class BlobTriggerExtension : IExtensionConfigProvider,
```

Comment:
nit: same comment as before on naming consistency. E.g. should this be BlobTriggerExtensionConfig? I can't see all the other extensions and the pattern they follow, but I don't think we're consistent across the extrensions.